docs/antrea-gke.md

```diff
@@ -0,0 +1,128 @@
+2. Access your cluster
+
+    ```bash
+    kubectl get nodes
+    NAME                                      STATUS   ROLES    AGE     VERSION
+    gke-cluster1-default-pool-93d7da1c-61z4   Ready    <none>   3m11s   v1.14.10-gke.17
+    gke-cluster1-default-pool-93d7da1c-rkbm   Ready    <none>   3m9s    v1.14.10-gke.17
+    ```
+
+3. Create a cluster-admin-binding
+
+    ```bash
+    kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user user@email.com
+    ```
+
+    Note: To create clusterRoleBinding, the user must have `container.clusterRoleBindings.create` permission.
+Use this command to enable it, if the previous command fails due to permission error. Only cluster Admin can
+assign this permission.
+
+    ```bash
+    gcloud projects add-iam-policy-binding $GKE_PROJECT --member user:user@email.com --role roles/container.admin
+    ```
+
+## Deploying Antrea
+
+1. Prepare the Cluster Nodes
+
+    Deploy antrea-node-init DaemonSet to enable kubelet to operate in CNI mode.
+
+    ```bash
+    kubectl apply -f https://github.com/vmware-tanzu/antrea/releases/download/v0.5.0/antrea-node-init.yml
+    ```
+
+2. Download and Update Antrea YAML
+
+    Download a released version of Antrea deployment YAML (>=v0.5.0). Update the `serviceCIDR` config in the
+antrea-gke.yml file with GKE_SERVICE_CIDR selected at the time of deploying GKE cluster.
+
+    ```bash
+    export TAG=v0.5.0
+    curl https://github.com/vmware-tanzu/antrea/releases/download/<TAG>/antrea-gke.yml --output antrea-gke.yml
+
+    cat antrea-gke.yml
+    ...
+        # CIDR Range for services in cluster. It's required to support egress network policy, should
+        # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
+        serviceCIDR: 10.94.0.0/16
+    ...
+    ```
+
+3. Deploy Antrea
+
+    Deploy Antrea using `kubectl apply -f antrea-gke.yml`. It will deploy single replica of Antrea controller
+and Antrea agent to every worker nodes of the GKE cluster. After a successful deployment you should be able to see
+these Pods running in your cluster:
+
+    ```bash
+    $ kubectl get pods --namespace kube-system  -l app=antrea -o wide
+    NAME                                READY   STATUS    RESTARTS   AGE   IP              NODE                                      NOMINATED NODE   READINESS GATES
+    antrea-agent-24vwr                  2/2     Running   0          46s   10.138.15.209   gke-cluster1-default-pool-93d7da1c-rkbm   <none>           <none>
+    antrea-agent-7dlcp                  2/2     Running   0          46s   10.138.15.206   gke-cluster1-default-pool-9ba12cea-wjzn   <none>           <none>
+    antrea-controller-5f9985c59-5crt6   1/1     Running   0          46s   10.138.15.209   gke-cluster1-default-pool-93d7da1c-rkbm   <none>           <none>
+    ```
+
+4. Restart remaining pods
+
+    Once Antrea is up and running, restart all pods in kube-system so they can be managed by Antrea.
```

Comment:
is it necessary to mention only kube-system namespace pods?

Reply:
I mean the command removes pods from kube-system and this is fresh installation. Assuming there are no other namespaces. 
Keep in mind, this is temporary. Once we integrate with GKE, this wont be needed at all.

Reply:
true .. but do we see a scenario where in the user already had a existing kubenet cluster and skips to these steps to replace with Antrea? i don't think there is any harm to say restart all pre-existing Pods and then perhaps in example keep the kube-system namespace